conf/pipeline/raredisease/hasta.config

```diff
@@ -1,10 +1,27 @@
 process {
-    
+
     withName:'PICARD_MARKDUPLICATES' {
         memory = { check_max( 90.GB  * task.attempt, 'memory' ) }
     }
     withName:'DEEPVARIANT' {
         cpus   = { check_max( 16      * task.attempt, 'cpus' ) }
         memory = { check_max( 80.GB   * task.attempt, 'memory' ) }
     }
-}
\ No newline at end of file
+    withName: BCFTOOLS_VIEW {
+        if (params.genome == 'GRCh37') {
+            ext.args = '--output-type v --apply-filters PASS --exclude "INFO/clinical_genomics_mipAF > 0.40 | INFO/swegenAF > 0.40 | INFO/clingen_ngiAF > 0.40 | INFO/gnomad_svAF > 0.40 "'
```

Comment:
Looks like the prior step outputs a gzipped vcf. Would be good to do this here as well.